internal/Get-Connection.ps1

```diff
@@ -0,0 +1,115 @@
+﻿Function Get-Connection
```

Comment:
I named it like this because it will replace Test-Connection in most functions. I thought that would read easy, and make clear this is the step where we test the connection and collect the IP and names to work with. Whether we resolve or grow or hammer the result in it's shape is not important to the user.
But you're right it's not really the connection, it's indeed only the info.

@RamblingCookieMonster wrote this: https://gallery.technet.microsoft.com/scriptcenter/Get-SPN-Get-Service-3bd5524a

Should this be one function? Maybe if 1 SPN for a computer is selected. Would you always need the Host or RestrictedKerberosHost? or the WSMAN?
I think we should keep it separate if you could need multiple SPN's, or if we need another function anyway to get computerinfo from AD. 

Maybe it can go in Test-SqlConnection too?